src/main.c

```diff
@@ -334,6 +350,55 @@ static void setup_adc(void)
 
 
 /**
+ * @brief TIM1 Interruption routine
+ *
+ *- Update event interrupt flag management
+ *- Led toggling each second
```

Comment:
Stops/dots after the sentences? :stuck_out_tongue_winking_eye: 

I think you are missing a space between `*` and the `-` in both list items. :blush:

Reply:
LED is an achronym.

Reply:
ok